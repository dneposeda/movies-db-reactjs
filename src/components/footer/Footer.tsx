import React from 'react';
import Logo from "../logo/Logo";

const Footer = () => (
        <div className="wrapper-footer">
            <div className="container">
                <div className="row">
                    <div className="col d-flex justify-content-center">
                        <Logo />
                    </div>
                </div>
            </div>
        </div>
    )

export default Footer;
