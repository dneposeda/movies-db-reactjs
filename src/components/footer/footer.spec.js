import React from 'react';
import Footer from './Footer';
import { mount } from 'enzyme';

describe('Footer', () => {
    it('renders correctly', () => {
        const component = mount (
            <Footer />
        );
        expect(component).toMatchSnapshot();
        expect(component.find('.logo')).toHaveLength(1);
    });
});
