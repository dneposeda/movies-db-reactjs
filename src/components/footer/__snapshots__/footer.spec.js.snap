// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<Footer>
  <div
    className="wrapper-footer"
  >
    <div
      className="container"
    >
      <div
        className="row"
      >
        <div
          className="col d-flex justify-content-center"
        >
          <Logo>
            <div
              className="logo"
            >
              <a
                className="logo__link"
                href="/"
              >
                <span
                  className="logo__text"
                >
                  <span>
                    netflix
                  </span>
                  roulette
                </span>
              </a>
            </div>
          </Logo>
        </div>
      </div>
    </div>
  </div>
</Footer>
`;
